08-rails/active-record-models.md

```diff
@@ -5,137 +5,133 @@
 - Begin exploring the AR _query interface_
 
 ## Active Record as a DSL
-Given a table in our database called _students_ that has this structure:
+Given a table in our database called _books_ that has this structure:
 
-| id | name     | pie         | birthday   |
+| id | title     | author         | description   |
 |:--:|:--------:|:-----------:|:----------:|
-|  1 | Sheena   | peach       | 08-13-1977 |
-|  2 | Rosa     | apple       | 05-01-2008 |
-|  3 | Raquel   | apple       | 03-27-2003 |
+|  1 | 'Hidden Figures'   | 'Margot Lee Shetterly'       | 'Good book that came before the movie' |
+|  2 | 'Practical Object-Oriented Design in Ruby'     | 'Sandi Metz'       | 'Good programming book' |
+|  3 | 'Kindred'   | 'Octavia E. Butler'       | 'Good sci-fi book' |
 
-We should have a corresponding file called `app/models/student.rb`. It will look like this:
+We should have a corresponding file called `app/models/book.rb`. It will look like this:
 
 ```ruby
-class Student < ActiveRecord::Base
+class Book < ActiveRecord::Base
 
 end
 ```
 
-This is our Student _model_, and it is connected to a table in the database called _students_. We don't have to explicitly tell Rails about that connection; by following the naming conventions (thank you, Inflector!), Rails knows which database table to use with which _model_. By doing this you'll also have the ability to map the columns of each row in that table with the attributes of the instances of your model.
+This is our Book _model_, and it is connected to a table in the database called _books_. We don't have to explicitly tell Rails about that connection; by following the naming conventions (thank you, Inflector!), Rails knows which database table to use with which _model_. By doing this you'll also have the ability to map the columns of each row in that table with the attributes of the instances of your model.
 
 For right now, our _models_ will likely be empty Ruby classes. The inherited class (`ActiveRecord::Base`) provides a tremendous amount of functionality. As we move forward in the curriculum, we will begin filling out our models with additional functionality.
```

Comment:
We could say "The inherited class (`ActiveRecord::Base` through `ApplicationRecord`) ..." here to be more accurate for Rails 5.